docs/blog/newsletter-june-2022.rst

```diff
@@ -0,0 +1,129 @@
+.. post:: June 01, 2022
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – June 2022
+#########################################
+
+Hi from your new newsletter editor, Aaron Collier. I'm taking over as editor this month from Beth Aitman and hoping to continue the doses of news and general awesomeness she and the other newsletter volunteers having been regularly bringing to your inboxes.
+
+I certainly have learned a lot from her already on how to make a newsletter useful and interesting. But it's also important to recognize that the heart of the newsletter comes from you, the Write the Docs community. I had a conversation in Slack recently with a long-standing member who didn't realize they could suggest stories to be included.
+
+So as I start this journey, I'd like to (re-)invite you to come with me. If you see a conversation in Slack you think would be great for the newsletter, tag it with the ``:suggest-for-newsletter:`` emoji. We'll round up the most interesting and thought-provoking ideas and bring them to you.
+
+The past month was full of goodness outside the newsletter as we wrapped up the Portland conference. Those who were there talked a lot about how great all the talks and unconference sessions were. For those of us who couldn't make it, keep an eye on the `Write the Docs YouTube channel <https://www.youtube.com/channel/UCr019846MitZUEhc6apDdcQ>`__ to at least see the talks.
```

Comment:
This is out now! https://www.youtube.com/playlist?list=PLZAeFn6dfHpnDhFvXG8GprqlLlzSQRBui